src/basic/path-util.c

```diff
@@ -752,38 +752,6 @@ int fsck_exists(const char *fstype) {
         return executable_is_good(checker);
```

Comment:
Typo in commit message: parse-helper.c -> parse-argument.c

Reply:
Oh, I renamed the file but forgot to update the commit message. Let's leave it like this, unless there's another reason for update.

Reply:
Yeah.

Reply:
Fixed now.